homeassistant/components/reolink/__init__.py

```diff
@@ -40,16 +41,20 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
 
     try:
         if not await host.async_init():
+            await host.stop()
             raise ConfigEntryNotReady(
                 f"Error while trying to setup {host.api.host}:{host.api.port}: "
                 "failed to obtain data from device."
             )
+    except UserNotAdmin as err:
+        raise ConfigEntryAuthFailed(err) from UserNotAdmin
```

Comment:
Should this not have been `from err`

Reply:
yes I have changed it in a commit to this PR: https://github.com/home-assistant/core/pull/85654
I thought it was to small to open a seperate PR.